### ISSUE TITLE
ImportError: The _imaging C module is not installed

### ISSUE THREAD

**Issue by timonoj** [haiwen/seafile-rpi#11]
Hi guys,

Thanks for the hard work with this little piece of awesomeness! I wanted to raise an issue I seem to be having since I upgraded to seafile server 5.0.4: Seahub seems not to be working properly, so I get an error page. When checking seahub_django_request.log, I get the following:

```
    import ImageFont
  File "/usr/lib/python2.7/dist-packages/PILcompat/ImageFont.py", line 1, in <module>
    from PIL.ImageFont import *
  File "/media/seafile/bananapi/seafile-server-5.0.4/seahub/thirdpart/PIL-1.1.6-py2.7-linux-armv7l.egg/PIL/ImageFont.py", line 115, in <module>
    class FreeTypeFont:
  File "/media/seafile/bananapi/seafile-server-5.0.4/seahub/thirdpart/PIL-1.1.6-py2.7-linux-armv7l.egg/PIL/ImageFont.py", line 135, in FreeTypeFont
    def getmask2(self, text, mode="", fill=Image.core.fill):
  File "/media/seafile/bananapi/seafile-server-5.0.4/seahub/thirdpart/PIL-1.1.6-py2.7-linux-armv7l.egg/PIL/Image.py", line 36, in __getattr__
    raise ImportError("The _imaging C module is not installed")
ImportError: The _imaging C module is not installed
```

Any ideas of what can be causing it? I didn't change my python install, it has been working for almost a year without issue.

**Comment by timonoj** [comment]
Sorry...fixed! 
I don't think I changed my libjpeg library, but anyway, after checking it seems libjpeg8 was installed, and after reverting to libjpeg62, it got back to work. So there's that...Why is this happening? Any changes done in the seafile end? Or my python decided to upgrade and I missed it?

**Comment by wapolinar** [comment]
Yes, I had exactly the same issue. After updating dev-python/pillow (using Gentoo on the RPI2) and removing the old imaging package, the issue went away.

Still strange that a minor version change (5.0.3->5.0.4) introduces these incompabilities.

**Comment by abysso2** [comment]
Hi,
the main difference is, that the 5.0.3 was built by jobenvil on a bananapi / weezy and i built the 5.0.4 on a cubietruck / jessie.

When i installed the python-library PIL, i had initially no jpeg-support, so i installed the 

libjpeg-dev
libjpeg62-turbo-dev

and linked the so-files to /usr/lib/

Afterwards, i compiled the PIL-package with the jpeg-support - but this is causing your issues ....

I will think about changing to libjpg8-dev the next time, okay?

Sorry for the inconvenience!

Regards,
A.

**Comment by abysso2** [comment]
Just found this here:

https://github.com/Automattic/node-canvas/issues/524

libjpeg8-dev is not available on Debian Jessie #524

**Comment by jobenvil** [comment]
Updated R5.0.4 packages to work since Wheezy.
